src/plugins/vis_type_timeseries/public/application/components/aggs/percentile_rank/multi_value_row.tsx

```diff
@@ -32,16 +35,33 @@ export const MultiValueRow = ({
   onAdd,
   disableAdd,
   disableDelete,
+  enableColorPicker,
 }: MultiValueRowProps) => {
   const onFieldNumberChange = (event: ChangeEvent<HTMLInputElement>) =>
     onChange({
       ...model,
       value: get(event, 'target.value'),
     });
 
+  const onColorPickerChange = (props: ColorProps) =>
+    onChange({
+      ...model,
+      color: props.color || '#68BC00',
```

Comment:
nitpick: this is used inline 4 times across the files, maybe we could store is as DEFAULT_COLOR or something?